src/surveys/components/CTAResponse.tsx

```diff
@@ -0,0 +1,63 @@
+import { h } from 'preact';
+import { CTASettings, CTAType, Question, Theme } from '../../types';
+import { Button } from '../../components/Button/Button';
+import { Header } from '../../components';
+
+type Props = {
+  question: Question;
+  theme?: Theme;
+  onAnswered: () => void;
+};
+
+export const CTAResponse = ({ question, theme, onAnswered }: Props) => {
+  const onClickHandler = () => {
+    if (onAnswered) {
+      onAnswered();
+    }
+  };
+
+  return (
+    <div className={'border'}>
+      <Header
+        title={question?.label}
+        description={question?.description}
+        color={theme?.question ? theme?.question : '#050505'}
+      />
+
+      {(question?.settings as CTASettings).type === CTAType.CLOSE && (
+        <div>
+          <Button
```

Comment:
@afolasope, why do we have 3 buttons that all look the same?

Reply:
@afolasope I feel we should just return `null` if type is `hidden`, then render the same button for other types.

Reply:
I have fixed it